From c20a32b8b83a60a02ce6a8df261f12604c1f727e Mon Sep 17 00:00:00 2001
From: Romain Naour <romain.naour@gmail.com>
Date: Fri, 5 Aug 2016 21:34:27 +0200
Subject: [PATCH] E: include uuid.h only when Wayland support is enabled.

libuuid is checked only when Wayland support is enabled and
uuid_t uuid is guarded by HAVE_WAYLAND.

So move include uuid.h below a HAVE_WAYLAND.

Signed-off-by: Romain Naour <romain.naour@gmail.com>

fix T4298
---
 src/bin/e_pixmap.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/bin/e_pixmap.c b/src/bin/e_pixmap.c
index febb689..72b26e8 100644
--- a/src/bin/e_pixmap.c
+++ b/src/bin/e_pixmap.c
@@ -2,6 +2,7 @@
 
 #ifdef HAVE_WAYLAND
 # include "e_comp_wl.h"
+# include <uuid.h>
 # ifndef EGL_TEXTURE_FORMAT
 #  define EGL_TEXTURE_FORMAT		0x3080
 # endif
@@ -18,7 +19,6 @@
 #endif
 
 #include <sys/mman.h>
-#include <uuid.h>
 
 static Eina_Hash *pixmaps[2] = {NULL};
 static Eina_Hash *aliases[2] = {NULL};
-- 
2.5.5

